### PR TITLE
drop support for nodejs 14 with controller 5 as it is EOL

### LINKED ISSUE #2239: Update README (add nodejs 18)
https://github.com/ioBroker/ioBroker.js-controller#compatibility

### PR DESCRIPTION
**Pull request by foxriver76**
- and adjust docs (closes #2239)
- Should we communicate in Telegram + Forum again? ;-)

### CODE REVIEW

File: .github/workflows/ci-tests.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [windows-latest, ubuntu-20.04, macos-latest]
-        node: [14, 16, 18]
+        node: [16, 18]
```
Comment (Apollon77) [inline .github/workflows/ci-tests.yml:46]: lets start also testing 20? maybe in a follow up PR?

File: README.md [hunk 2]
```diff
@@ -13,12 +13,13 @@ The ioBroker.js-controller is the heart of any ioBroker installation. The contro
 **Please check the js-controller compatibility information below which version runs on your Node.js version**
 
 ## Compatibility
-* js-controller 4.x works with Node.js 12.x, 14.x and 16.x (incl. up to NPM 8)
+* js-controller 5.x works with Node.js 16.x and 18.x
```
Comment (Apollon77) [inline README.md:16]: add "20 but untested" or such here already please

**Review by Apollon77 — APPROVED** [review]
... but we should inform devs again like last time some days before merge?

**Comment by foxriver76** [comment]
https://forum.iobroker.net/topic/64954/controller-v5-kein-support-von-node-js-14

+ multiple Telegram channels done

**Review by Apollon77 — APPROVED** [review]

**Pull request merged**